### DISCUSSION TITLE
Updating

### DISCUSSION THREAD

**Discussion by DiamondHead1022** [OrcaSlicer/OrcaSlicer#939] [Q&A]
Whenever I press the button to update it brings me to the GitHub and then I download that version but when I open up orca slicer it opens the old version

**Comment by fr3ak2402** [comment]
Hi @DiamondHead1022.

(Windows)
If you have downloaded new version, you can delete old version and paste the new version into the folder where the old version was originally. 

After that you just start the exe of the new version.

(macOS)
Delete the old version from the Applications folder and paste the downloaded version into the Applications folder.

After that you can open OrcaSlicer as usual and have the latest version.

**Reply by DiamondHead1022** [reply]
Will that save my settings?

**Reply by fr3ak2402** [reply]
Yes, your settings are not saved in this folder.

But there:
(Windows)
/User/Your Username/Appdata/Roaming/OrcaSlicer
(macOS)
/User/Your Username/Library/Application Support/OrcaSlicer

**Comment by buckeyestargazer** [comment]
It sure would be nice if OrcaSlicer would simply install like any other normal program.  That would avoid these kinds of confusion and make it far easier to add shortcuts to the start menu or desktop etc.

**Reply by fr3ak2402** [reply]
Yes, you are absolutely right. This is already the case with Mac, where the applications can be installed as usual. With Windows, this is really cumbersome.
